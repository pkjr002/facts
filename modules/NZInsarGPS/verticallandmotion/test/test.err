mv: cannot stat ‘*globalsl*’: No such file or directory
