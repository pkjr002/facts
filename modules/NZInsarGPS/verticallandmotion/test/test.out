Initiating test in /scratch/rk509/test.1666634089...
Extracting data files...
Executing workflow...
python NZInsarGPS_preprocess_verticallandmotion.py --pipeline_id verticallandmotion-NZInsarGPS-verticallandmotion
python NZInsarGPS_fit_verticallandmotion.py --pipeline_id verticallandmotion-NZInsarGPS-verticallandmotion
python NZInsarGPS_project_verticallandmotion.py --pipeline_id verticallandmotion-NZInsarGPS-verticallandmotion
python NZInsarGPS_postprocess_verticallandmotion.py --pipeline_id verticallandmotion-NZInsarGPS-verticallandmotion --nsamps 20000 --seed 7331 --baseyear 2005 --pyear_start 2020 --pyear_end 2300 --pyear_step 10 --locationfile test/location_list.lst
Collecting output files...
